src/EntityFramework.SqlServer/SqlGen/SqlGenerator.cs

```diff
@@ -2639,6 +2648,38 @@ public override ISqlFragment Visit(DbSkipExpression e)
             return result;
         }
 
+        /// <summary>
```

Comment:
I think we only use actual XML doc comments in public surface, so these should be `//` instead of `///`.